thirdparty/patches/avro-1.10.2.c.patch

```diff
@@ -41,6 +41,20 @@
 -    set(SNAPPY_LIBRARIES "")
 -    message("Disabled snappy codec. libsnappy not found or zlib not found.")
 -endif (SNAPPY_FOUND AND ZLIB_FOUND)
+-
+-find_package(PkgConfig)
+-pkg_check_modules(LZMA liblzma)
+-if (LZMA_FOUND)
+-    set(LZMA_PKG liblzma)
+-    add_definitions(-DLZMA_CODEC)
+-    include_directories(${LZMA_INCLUDE_DIRS})
+-    link_directories(${LZMA_LIBRARY_DIRS})
+-    message("Enabled lzma codec")
+-else (LZMA_FOUND)
+-    set(LZMA_PKG "")
+-    set(LZMA_LIBRARIES "")
+-    message("Disabled lzma codec. liblzma not found.")
+-endif (LZMA_FOUND)
 +# find_package(Snappy)
```

Comment:
If the code is not necessary, you can remove it.

Reply:
> If the code is not necessary, you can remove it.

updated